src/elasticodbc/dlg_specific.c

```diff
@@ -142,11 +142,15 @@ BOOL copyConnAttributes(ConnInfo *ci, const char *attribute,
         STRCPY_FIXED(ci->drivername, value);
     else if (stricmp(attribute, INI_SERVER) == 0)
         STRCPY_FIXED(ci->server, value);
+    else if (stricmp(attribute, INI_HOST) == 0)
```

Comment:
Do we want to combine this with the previous case, like in L149-50 and L152-3?

Reply:
Yes, will update this.